src/components/LHNOptionsList/OptionRowLHN.js

```diff
@@ -6,6 +6,7 @@ import {
     View,
     StyleSheet,
 } from 'react-native';
+import * as localStyles from './styles';
```

Comment:
This is looking good, but I think we should move `localStyles` into `styles/getOptionRowStyles`, similar to how `getModalStyles` is implemented

Reply:
Fixed.